extensions/wikia/DMCARequest/ChillingEffectsClient.class.php

```diff
@@ -1,21 +1,146 @@
 <?php
 
+/**
+ * Client class for sending notices to the Chilling Effects API.
+ *
+ * @author grunny
+ */
+
 namespace DMCARequest;
 
 class ChillingEffectsClient {
-	const BASE_URL = 'https://chillingeffects.org/notices';
+
+	const DMCA_TYPE = 'Dmca';
+	const DMCA_TOPIC = 5;
+
+	const SUCCESSFUL_STATUS = 201;
+
+	private $baseUrl;
 	private $apiToken;
 
-	public function __construct( $apiToken ) {
+	/**
+	 * Create client object.
+	 *
+	 * @param string $baseUrl  API URL.
+	 * @param string $apiToken API token.
+	 */
+	public function __construct( $baseUrl = '', $apiToken = '' ) {
```

Comment:
Ah, yeah, that's just left over from me being lazy while testing the methods that didn't need it at the command line. ;)